.github/workflows/build.yml

```diff
@@ -9,6 +9,7 @@ on:
   push: 
     branches:
       - main
+  pull_request:
```

Comment:
Why did I make this a separate ticket??? I should have realized how trivial it would be 😆 .

Reply:
haha... That is why I thought we were going to need (For some reason I didn't know yet) another yml file.